shell/platform/tizen/external_texture_surface_gl.cc

```diff
@@ -55,20 +55,20 @@ bool ExternalTextureSurfaceGL::PopulateTexture(
   const FlutterDesktopGpuBuffer* gpu_buffer =
       texture_callback_(width, height, user_data_);
   if (!gpu_buffer) {
-    FT_LOGD("[texture id:%ld] gpu_buffer is null", texture_id_);
+    FT_LOGE("[texture id:%ld] gpu_buffer is null", texture_id_);
```

Comment:
@xiaowei-guan The use of `FT_LOGD` wasn't intentional, right? Can we safely change these to `FT_LOGE`?

Reply:
we don't need change to FT_LOGE. Because In some cases it is normal to return null,such as when the application UI is displayed for the first time, this callback will also be tiggered.